src/components/explore/ProbeExplorer.svelte

```diff
@@ -241,12 +251,20 @@ $: if (hoverValue.x) {
               (explorerComparisonSmallMultiple.width
               - explorerComparisonSmallMultiple.left
               - explorerComparisonSmallMultiple.right) / 2 - 5]}
-            areaColor="var(--digital-blue-400)"
-            lineColor="var(--digital-blue-500)"
-          />
+            areaColor="var(--pantone-red-400)"
+            lineColor="var(--pantone-red-500)"
+          /> -->
         {/if}
-        {#if reference}
-          <Violin
+        {#if reference && reference[densityMetricType]}
+              <AdHocViolin start={(lp + rp) / 2} density={reference[densityMetricType]}
+                width={
+                  (explorerComparisonSmallMultiple.width
+                - explorerComparisonSmallMultiple.left
+                - explorerComparisonSmallMultiple.right) / 2 - 7} />
```

Comment:
I think the magic constant of 7 is throwing me off a bit. What's the significance of that?

Reply:
eyeballing mostly

Reply:
I'll move thse into variable names